### PR TITLE
add debug.log location override in wpconfig

### LINKED ISSUE #4069: Configuring wp-config.php Doc Update to add debug.log location override
Re: [Configuring wp-config.php](https://pantheon.io/docs/wp-config-php/)

Priority: Low

## Issue Description:
WordPress has an option to write debugging information to /wp-content/debug.log by default which is not writable by default in Pantheon. (https://codex.wordpress.org/Debugging_in_WordPress#WP_DEBUG_LOG)

## Suggested Resolution
Add relevant information in docs on how to override it and make it work in the platform across all environments

### PR DESCRIPTION
**Pull request by carl-alberto**
Closes #4069

## Effect
PR includes the following changes:
- Add relevant information in docs on how to enable it and how to make it work in the platform across all environments

## Remaining Work
- [ ] List any outstanding todos
- [ ] If needed

## Post Launch
To be completed by the docs team upon merge: 
- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Archive from **Done** in Waffle

### CODE REVIEW

**Comment by alexfornuto** [comment]
@carl-alberto a few questions:

 - Why not include this in the codeblock from the logs file? I've moved it into the docs from the gist, so it's easier to edit if that was the concern.
 - It seems like people don't recommend enabling this logging on production sites, which is what this allows. Should a warning be included?

**Comment by carl-alberto** [comment]
@alexfornuto 

> Why not include this in the codeblock from the logs file? I've moved it into the docs from the gist, so it's easier to edit if that was the concern.

I think this will also help. 

> It seems like people don't recommend enabling this logging on production sites, which is what this allows. Should a warning be included?

Yes, it is unusual to debug in live but I have encountered a customer trying to debug their non-working wp-cron pulling some live data from an external service and seems the only way to do that is setting up the wp debug log to a writable path, good idea in putting a warning so they will be reminded to turn that feature off once they are done with debugging as it has dangerous implications.

**Comment by alexfornuto** [comment]
@carl-alberto please review my latest changes to this PR.

File: source/_docs/logs.md [hunk 1]
```diff
@@ -196,9 +196,40 @@ No, access to Apache Solr logs is not available. For more information on debuggi
...
+  }
+}
+```
+
+By default, the WordPress debug log path is set to `/wp-content/` and not writable on Test or Live environments. This can be overridden to the <a href="/docs/wp-config-php/#how-do-i-change-the-default-debuglog-location" data-proofer-ignore>`/wp-content/uploads/` folder</a>.
+
+
+<div class="alert alert-danger" role="alert" markdown="1">
+#### Warning {.info}
+Enabling debug logging is not recommended for production environments due to the increased resource overhead. If you need to enable debug logging on the live environment to resolve a problem, remember to disable it when you are done.
```
Comment (carl-alberto) [inline source/_docs/logs.md]: Maybe we can add @alexfornuto ?

Enabling debug logging is not recommended for production environments due to the increased resource overhead and security risk.

**Comment by carl-alberto** [comment]
@alexfornuto Added some improvements in the wp-config.php snippet, other than that, all looks to me. :)

**Comment by alexfornuto** [comment]
@carl-alberto who would you suggest for technical review, to confirm that your code snippets are correct?

**Comment by carl-alberto** [comment]
I believe @geraldvillorente worked with a similar issue before and used a similar snippet or if @eabquina has some spare minutes to verify if that works on their end :)

**Comment by alexfornuto** [comment]
Thanks @oseldman!

**Comment by alexfornuto** [comment]
+1 :shipit:

**Pull request merged**